backend/src/test/java/com/htecgroup/skynest/utils/BucketEntityUtil.java

```diff
@@ -0,0 +1,10 @@
+package com.htecgroup.skynest.utils;
+
+import com.htecgroup.skynest.model.entity.BucketEntity;
+
+public class BucketEntityUtil extends BucketBasicUtil {
+
```

Comment:
I am wondering, if you just extend this BucketBasicUtil for property access, why don't you move them inside this class, and remove the BucketBasicUtil.. 

Also make this class final and with private constructor, because that is the usage of Util class, not extendable and closed for creating objects

Reply:
make sense, I've changed it like that